### DISCUSSION TITLE
Is FastAPI Beta, or Production?

### DISCUSSION THREAD

**Discussion by adrian-pc-code** [fastapi/fastapi#11816] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
NA
```

### Description

I see in various places online that FastAPI is considered beta such as:
https://pypi.org/project/fastapi/

```
Classifiers
Development Status
[4 - Beta](https://pypi.org/search/?c=Development+Status+%3A%3A+4+-+Beta)
```

But in the official website it says: "Robust: Get production-ready code. With automatic interactive documentation."

Is 0.111 ready for production?

### Operating System

Other

### Operating System Details

NA

### FastAPI Version

0.111

### Pydantic Version

NA

### Python Version

NA

**Comment by YuriiMotov** [comment, marked as answer]
The development status is Beta and it means that braking changes can be made at any time.
But as I know a lot of companies use FastAPI in production
